### ISSUE TITLE
Crashes not reported, Fabric error

### ISSUE THREAD

**Issue by RobertGardner** [EddyVerbruggen/nativescript-plugin-firebase#1276]
I have been using the firebase plugin since one of the first versions. (It's awesome!) I recently switched from Crash Reporting to Crashlytics and can't get the crashes to report correctly. It's been over a day since I switched, set it up in the console, and forced a crash, but the Firebase Crashlytics console still says it's waiting for the first crash report.

Whenever my app starts, I get the following error in the console (on iOS running on Mac in an emulator):
```
[Fabric] failed to download settings Error Domain=FABNetworkError Code=-5 "(null)" UserInfo={status_code=403, type=2, request_id=d382b7f4e2e646d038b37b0c0ff3db66, content_type=application/json; charset=utf-8}
```

I suspect this is the reason it's not working for me, but I don't know how to fix it. I found some instructions on Stackoverflow about updating a key in `Info.plist` but I'm not sure where to find the API Key and don't see any instructions here about this, so I'm not sure that will fix it.

(I see other bugs here about reports not showing up and have tried all suggestions, but no bug reports mention FABNetworkError. So I think this is a different issue.)

**Comment by EddyVerbruggen** [comment]
I have no conclusive answer for you, but regarding the plist stuff, make sure you have the latest [GoogleService-Info.plist](https://github.com/EddyVerbruggen/nativescript-plugin-firebase#prerequisites) in your project.

Can you link to that SO post?

**Comment by RobertGardner** [comment]
[SO post](https://stackoverflow.com/questions/50269731/fabric-failed-to-download-settings-error-domain-fabnetworkerror-code-5) about this. Note: I followed the instructions in that post and it doesn't build. XCode reports: `Mismatched configuration: Fabric entry found in app's Info.plist, but no API Key provided to the run script. If this app was set up in Fabric, or linked to Firebase, pass your API key to the run script as the first argument. If this app was onboarded via Firebase, remove the Fabric entry in your Info.plist`

I updated `GoogleService-Info.plist`. There were no substantial differences. It still fails with the same problem. I don't see anything in that file about crashlytics or crash reporting, but I do see my API Key. To recap, I get an error in the console with `Error Domain=FABNetworkError` and when I execute `crashlytics.crash()` the app crashes but nothing is logged to the console and even days later nothing is reported in the Firebase console. (I stepped into the plugin code and verified that crashlytics is enabled.)

I'm not sure it's relevant, but I'm now getting a whole bunch of warnings when my app launches regarding Fabric that look similar to:
```
Class FABNetworkClient [and many other FAB/CLS classes] is implemented in both <long path>/MyApp.app/Frameworks/TNSCrashlyticsLogger.framework/TNSCrashlyticsLogger (0x107dc6d78) and <same path>/MyApp.app/MyApp (0x102446e78). One of the two will be used. Which one is undefined.
```
Though they're annoying, I'm assuming they don't mean anything important and can be ignored.

**Comment by gt6089** [comment]
I am also getting a similar error trying to set up Crashlytics on iOS for the first time.

`[Fabric] failed to download settings Error Domain=FABNetworkError Code=-5 "(null)" UserInfo={status_code=403, type=2, request_id=, content_type=application/json; charset=utf-8}`

- I have the latest version of the plugin.
- I have the latest GoogleService-Info.plist.
- I'm using a real iOS device.
- Android works fine; seeing forced crashes show up on the dashboard.

**Comment by RobertGardner** [comment]
Well, it's almost embarrassing to admit what was causing my problem... ;-)

I had gone through the steps of clicking "set up Crashlytics for the first time" on my Android project, not on my iOS project. I just did it on iOS and the crash was eventually reported.

Note that to see it I had to upload dSYMs, which was an incredible pain. For those interested, I finally found the upload script and the dSYM files. They were in the `platforms` folder of my app.

I ran this script to upload:
```
platforms/ios/Pods/Crashlytics/iOS/Crashlytics.framework/upload-symbols -gsp platforms/ios/<app_name>/Resources/GoogleService-Info.plist -p ios platforms/ios/build/Debug-iphonesimulator/<app_name>.app.dSYM
```
